dotcom-rendering/src/layouts/PictureLayout.tsx

```diff
@@ -265,122 +278,142 @@ export const PictureLayout = ({ article, NAV, format }: Props) => {
+							showTopBorder={false}
 							padSides={false}
-							element="aside"
+							backgroundColour={brandBackground.primary}
+							element="nav"
 							format={format}
-							showTopBorder={showSubNavTopBorder}
 						>
-							<Island defer={{ until: 'idle' }}>
-								<SubNav
-									subNavSections={NAV.subNavSections}
-									currentNavLink={NAV.currentNavLink}
-									linkHoverColour={
-										palette.text.articleLinkHover
-									}
-									borderColour={palette.border.subNav}
-									subNavLinkColour={palette.text.subNavLink}
-								/>
-							</Island>
+							<Nav
+								nav={props.NAV}
+								isImmersive={
+									format.display === ArticleDisplay.Immersive
+								}
+								displayRoundel={
+									format.display ===
+										ArticleDisplay.Immersive ||
+									format.theme === ArticleSpecial.Labs
+								}
+								selectedPillar={props.NAV.selectedPillar}
+								subscribeUrl={
+									article.nav.readerRevenueLinks.header
+										.subscribe
+								}
+								editionId={article.editionId}
+								headerTopBarSwitch={
+									!!article.config.switches.headerTopNav
+								}
+							/>
 						</Section>
-					)}
 
-					<Section
-						fullWidth={true}
-						backgroundColour={palette.background.article}
-						padSides={false}
-						showTopBorder={false}
-						borderColour={palette.border.secondary}
-					>
-						<StraightLines
-							count={4}
-							color={palette.border.secondary}
-							cssOverrides={css`
-								display: block;
-							`}
-						/>
-					</Section>
-				</SendToBack>
-			</div>
+						{props.NAV.subNavSections && (
+							<Section
+								fullWidth={true}
+								backgroundColour={palette.background.article}
+								padSides={false}
+								element="aside"
+								format={format}
+								showTopBorder={showSubNavTopBorder}
+							>
+								<Island defer={{ until: 'idle' }}>
+									<SubNav
+										subNavSections={
+											props.NAV.subNavSections
+										}
+										currentNavLink={
+											props.NAV.currentNavLink
+										}
+										linkHoverColour={
+											palette.text.articleLinkHover
+										}
+										borderColour={palette.border.subNav}
+										subNavLinkColour={
+											palette.text.subNavLink
+										}
+									/>
+								</Island>
+							</Section>
+						)}
+
+						<Section
+							fullWidth={true}
+							backgroundColour={palette.background.article}
+							padSides={false}
+							showTopBorder={false}
+							borderColour={palette.border.secondary}
+						>
+							<StraightLines
+								count={4}
+								color={palette.border.secondary}
+								cssOverrides={css`
+									display: block;
+								`}
+							/>
+						</Section>
+					</SendToBack>
+				</div>
+			)}
 
 			<main
 				data-layout="PictureLayout"
 				id="maincontent"
 				lang={decideLanguage(article.lang)}
 				dir={decideLanguageDirection(article.isRightToLeftLang)}
 			>
+				{renderingTarget === 'Apps' && (
```

Comment:
Sorry not super familiar with this ad code. How do we decide that we want an ad here?